src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -60,6 +57,12 @@
     Text bracketCloseText;
     Text plusText;
 
+    private static final String APPEND_TO_WATCH_LIST_FORMAT = "%s;%s#%d";
+    private static final String CREATE_WATCH_LIST_FORMAT = "id:%s#%d";
```

Comment:
Hmm, appending strings is a bit fragile, because the filter can be edited by the user. Could we do it on the AST level instead?

- Detection: traverse and find that there are only id qualifiers
- Create: create id qualifier
- Append: nest existing expression in disjunction

The first should be doable with `find`. We don't have a higher-level method yet so we can use that.